query/common/src/main/java/org/codice/ddf/admin/common/services/ServiceCommons.java

```diff
@@ -52,30 +54,31 @@ public String resolveProperty(String str) {
                 .collect(Collectors.toList());
     }
 
-    public static ReportImpl createManagedService(Map<String, Object> serviceProps, String factoryPid,
-            ConfiguratorFactory configuratorFactory) {
+    public static ReportImpl createManagedService(Map<String, Object> serviceProps,
+            String factoryPid, ConfiguratorFactory configuratorFactory,
```

Comment:
Can you leave a todo in this class to remove these static methods and pass the configuratorFactory through the constructor please